gocelery_test.go

```diff
@@ -1437,3 +1428,21 @@ func addMap(a, b map[string]interface{}) map[string]interface{} {
 	}
 	return c
 }
+
+func convertInterfaceToStringSlice(i interface{}) []string {
+	is := i.([]interface{})
+	stringSlice := make([]string, len(is))
+	for i, v := range is {
+		stringSlice[i] = fmt.Sprint(v)
```

Comment:
nitpick but I believe `fmt.Sprint` is a lot slower than string cast

Reply:
Perhaps, we can try to detect the type of the interface by feeding it back to convert method

Reply:
On the other hand, I think this is probably okay as is.